tee-worker/omni-executor/pumpx/src/pumpx_api/mod.rs

```diff
@@ -113,6 +113,7 @@ impl PumpxApi {
 			.post(endpoint)
 			.header("X-Language", language.unwrap_or("en".to_string()))
 			.bearer_auth(access_token)
+			.json(&serde_json::json!({}))
```

Comment:
If this needs to be merged please put a comment here so we know that this is something required and should not be removed in the future.